### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by Mohitk391**
Homepage added.
Basic files setup is also done.

Files to check: 
1) Homepage.js
2) homepage.css
3) NavBar.js
4) Navbar.css

preview:  https://62d70ed8bd3cd9142ad8e84e--watch-easy-homepage.netlify.app/

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,41 +1,19 @@
...
-            target="_blank"
-            rel="noreferrer"
-          >
-            Contribute
-          </a>
-        </div>
-      </header>
+      <Routes>
+        <Route path="/" element={<HomePage />} />
+        <Route path="/explore" element={ <HomePage /> } />
```
Comment (MayankKumar10) [inline src/App.js:10]: You have added HomePage Elements in All route there should be different components in every route.

File: src/components/NavBar/NavBar.css [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+    min-width: max-content;
+    font-size: 1rem;
+    background-color: var(--bg-off-white-1);
+    box-shadow: 0px 8px 16px 0px var(--bg-box-shadow-1);
+    z-index: 1;
+}
+
+.dropdown-content > * {
+    color: var(--black);
+    padding: 12px 16px;
```
Comment (MayankKumar10) [inline src/components/NavBar/NavBar.css:31]: you can use rem here instead of px

**Review by MayankKumar10 — APPROVED** [review]
Look into these Suggestions everything else looks good to me you can merge it.

**Review by SomeEarth86 — APPROVED** [review]
Looks good to merge.

**Pull request merged**